src/components/InitialCountForm.component.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class InitialCountForm extends Component {
+  state = { submitted: false, valid: true }
```

Comment:
same as above, use a constructor for initial state

Reply:
same here